lib/tasks/dial.js

```diff
@@ -196,7 +196,11 @@ class TaskDial extends Task {
       else {
         this.epOther = cs.ep;
         if (this.dialMusic && this.epOther && this.epOther.connected) {
-          this.epOther.play(this.dialMusic).catch((err) => {});
+          (async() => {
```

Comment:
what happens if an error is thrown from the play command?  Will it be an uncaught exception and crash the program?  This would need to be tested

Reply:
The exception is being handled by the catch section attached to play method. The main issue could be if the play method will be throwing the exception in a loop and we will end with endless loop blocking all other tasks.

Reply:
Not sure that this is a realistic scenario

Reply:
Maybe we could wait inside the catch block to workaround this

Reply:
@davehorton I've added a short delay inside the catch section, just in case we receive a malformed url for dialMusic.